app/main.py

```diff
@@ -1,3 +1,80 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+from app.additional_functions import (calculate_distance,
+                                      calculate_product_cost,
+                                      purchase_and_print_receipt)
+
+
+def shop_trip() -> None:
```

Comment:
it is a really big function, try to divide into a few function, and it would be better to  write them as methods in classes